rero_ils/accounts_views.py

```diff
@@ -139,3 +199,15 @@ def post(self, args):
             self.verify_password(**args)
             self.change_password(**args)
         return self.success_response()
+
+
+class DisabledAuthApiView(MethodView):
+    """Disabled Account rest auth api."""
+
+    def post(self, **kwargs):
+        """Post Data."""
+        abort(404)
+
+    def get(self, **kwargs):
+        """Get Data."""
+        abort(404)
```

Comment:
Suggestion : You could write 
```python
    def post(self, **kwargs):
        """Post Data."""
        abort(404)

    get = post
```
in this case, the GET and POST method are always the same.